docker/install_plugin_binary.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo "Building the Terraform provider plugin binary..."
+make build
+mkdir -p /root/.terraform.d/plugins/linux_amd64
+cp /go/bin/terraform-provider-citrixitm /root/.terraform.d/plugins/linux_amd64/
```

Comment:
I think an explicit line break here is 👍 
`cp /go/bin ... \
  /root/.terraform`

Reply:
Done.